src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/DefaultTagHelperCompletionServiceTest.cs

```diff
@@ -153,7 +159,7 @@ public void StringifyAttributes_DirectiveAttribute()
             var startTag = (MarkupTagHelperStartTagSyntax)syntaxTree.Root.LocateOwner(sourceChangeLocation).Parent;
 
             // Act
-            var attributes = DefaultTagHelperCompletionService.StringifyAttributes(startTag.Attributes);
+            var attributes = TagHelperFactsService.StringifyAttributes(startTag.Attributes);
```

Comment:
All of these `TagHelperFactsService.StringifyAttributes` tests need to move to the TagHelperFactsService specific tests